### DISCUSSION TITLE
[QUESTION] About Microsoft.Maui.Graphics

### DISCUSSION THREAD

**Discussion by sharpwood** [mono/SkiaSharp#1778] [Q&A] [answered]
Microsoft.Maui.Graphics contains some functions similar to SkiaSharp. What is the relationship between the two products and can they replace each other?

**Comment by nor0x** [comment, marked as answer]
`Maui.Graphics` is an abstraction of graphic APIs that support various backends to run on many platforms. For example `CoreGraphics` on Apple platforms or `SharpDX` on Windows. In addition to these native backends there is also a `SkiaSharp` backend on most platforms. So the idea is to have one API which you use to write your rendering code and run it on a variety of platforms either backed by `SkiaSharp` or the built-in engine of the platform.

Here you can find an overview of the abstractions
[https://github.com/dotnet/Microsoft.Maui.Graphics#supported-platforms](https://github.com/dotnet/Microsoft.Maui.Graphics#supported-platforms)

**Reply by mattleibow** [reply]
Thanks for the write up! The nice thing about that table is that SkiaSharp is everywhere ;)
